code/modules/events/spacevine.dm

```diff
@@ -586,9 +586,13 @@
 		if(prob(mutativeness))
 			var/datum/spacevine_mutation/random_mutate = pick_weight(vine_mutations_list - vine.mutations)
 			var/total_severity = random_mutate.severity
+			var/mutation_is_valid = TRUE
 			for(var/datum/spacevine_mutation/mutation as anything in vine.mutations)
+				if(random_mutate.type == mutation.type) // see #68821, mutations from parent.mutations can have a different address than those in vine_mutations_list
```

Comment:
The explanation given in this PR doesn't elaborate much on the one you put in this code comment, might be worthwile to drop the external reference entirely and write a better note. GitHub will eventually shitcan us, it's a matter of time. May as well make the code as good as we can.